dbt_modules/snowplow/dbt_project.yml

```diff
@@ -0,0 +1,32 @@
+#settings specifically for this models directory
+#config other dbt settings within ~/.dbt/profiles.yml
+name: 'snowplow'
+version: '0.0.1'
+
+source-paths: ["models"]
+target-path: "target"
+clean-targets: ["target"]
+test-paths: ["test"]
+analysis-paths: ["analysis"]
+data-paths: ["data"]
+macro-paths: ["macros"]
+
+models:
+  snowplow:
+    base:
+      materialized: ephemeral
```

Comment:
@fkalkhalidi you want to make changes within your project dbt_project.yml file, not in the module. https://github.com/perfectketo/dbt/blob/master/dbt_project.yml

timezones are something I believe Jacob looked into when implementing this. it is possible that you want to change from PST to UTC in your project so that all warehouse data is stored in utc

Reply:
Gotcha, ya I wanted to test it on my personal dbt branch to see if it would make a difference at all. Closing this PR.